src/_components/RoutesComponent.jsx

```diff
@@ -5,14 +5,15 @@ import { connect } from 'react-redux';
 import { loginAction } from '../_actions/login.action';
 
 import AuthenticateComponent from './AuthenticateComponent';
+import AssetTypesComponent from '../components/AssetTypesComponent'
 import LoginComponent from '../components/LoginComponent';
 import DashboardComponent from '../components/DashboardComponent';
 
 import getToken from '../_utils/getToken';
 
 class RoutesComponent extends React.Component {
   state = {
-    isAuthenticated: false,
+    isAuthenticated: true,
```

Comment:
Just curious why `isAuthenticated` has a default value of `true` in the state. Shouldn't it start out as `false` and change to `true` after an action is dispatched? e.g. login

Reply:
I did this to bypass an Authenticated bug being worked on by Hector. I have fixed it.